.gitignore

```diff
@@ -6,4 +6,5 @@ docs/_build/
 htmlcov
 dist
 .cache
-.hypothesis
\ No newline at end of file
+.hypothesis
+.venv*
```

Comment:
oh and please don't do this I don’t want put stuff there that doesn’t affect everyone.  do you know about `.git/info/exclude`?